lib/handlers/hard-quote/handler.ts

```diff
@@ -87,6 +88,19 @@ export class QuoteHandler extends APIGLambdaHandler<
     } else {
       cosignerData = getDefaultCosignerData(request);
       log.info({ cosignerData: cosignerData }, 'open order with default cosignerData');
+
+      // The RFQ responses are logged in getBestQuote() 
+      // we log the Open Orders here
+      log.info({
+        eventType: 'QuoteResponse',
+        body: { 
+          ...logBody,
+          amountIn: request.totalInputAmountStart.toString(),
+          amountOut: request.totalOutputAmountStart.toString(),
```

Comment:
@ConjunctiveNormalForm can you double check these should be the `amountIn` and `amountOut` values that we log for Open Orders?

Reply:
maybe better to use the response fields?

Reply:
Great call out. I didn't see that.